### PR TITLE
Feature/verix

### PR DESCRIPTION
**Pull request by sebeth-fa**

### CODE REVIEW

File: spec/rest/verix/verix_records_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+RSpec.describe FinApps::REST::VerixRecords do
+  include SpecHelpers::Client
+
+  # noinspection RubyBlockToMethodReference
+  let(:api_client) { client }
+
+  describe '#create' do
+    subject(:create) do
+      FinApps::REST::VerixRecords.new(api_client).create(
+          code: 'authcode',
```
[the remark this answers is not in the document]
Comment (qbantek) [reply spec/rest/verix/verix_records_spec.rb]: Fixed

File: lib/finapps/rest/verix/verix_records.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      def create(params)
+        super(params, 'v/record')
+      end
+
+      def list
+        super 'v/record'
+      end
+    end
+  end
+end
```
[the remark this answers is not in the document]
Comment (qbantek) [reply lib/finapps/rest/verix/verix_records.rb]: Fixed

File: lib/finapps/rest/verix/verix_records.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module FinApps
+  module REST
+    class VerixRecords < FinAppsCore::REST::Resources # :nodoc:
+
```
[the remark this answers is not in the document]
Comment (qbantek) [reply lib/finapps/rest/verix/verix_records.rb]: Fixed

File: lib/finapps/rest/verix/verix_metadata.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module FinApps
+  module REST
+    class VerixMetadata < FinAppsCore::REST::Resources # :nodoc:
+
```
[the remark this answers is not in the document]
Comment (qbantek) [reply lib/finapps/rest/verix/verix_metadata.rb]: Fixed

File: lib/finapps/rest/verix/verix_metadata.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+      def list
+        super 'v/metadata'
+      end
+    end
+  end
+end
```
[the remark this answers is not in the document]
Comment (qbantek) [reply lib/finapps/rest/verix/verix_metadata.rb]: Fixed

File: spec/rest/verix/verix_records_spec.rb [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    it('returns a consumer id') { expect(list[RESULTS].first).to have_key(:consumer_id) }
+    it('returns an error node') { expect(list[RESULTS].first).to have_key(:error) }
+    it('returns a list of documents') { expect(list[RESULTS].first).to have_key(:documents) }
+    it('returns a document type') { expect(list[RESULTS].first[:documents].first).to have_key(:type) }
+    it('returns a document id') { expect(list[RESULTS].first[:documents].first).to have_key(:document_id) }
+    it('returns a document count') { expect(list[RESULTS].first[:documents].first).to have_key(:count) }
+    it('returns a document downloaded bool') { expect(list[RESULTS].first[:documents].first).to have_key(:downloaded) }
+    it('returns a document date_synced') { expect(list[RESULTS].first[:documents].first).to have_key(:date_synced) }
+  end
+end
```
[the remark this answers is not in the document]
Comment (qbantek) [reply spec/rest/verix/verix_records_spec.rb]: Fixed

File: spec/rest/verix/verix_records_spec.rb [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+  # noinspection RubyBlockToMethodReference
+  let(:api_client) { client }
+
+  describe '#create' do
+    subject(:create) do
+      FinApps::REST::VerixRecords.new(api_client).create(
+          code: 'authcode',
+          download: [
+              'form_1040'
```
[the remark this answers is not in the document]
Comment (qbantek) [reply spec/rest/verix/verix_records_spec.rb]: Fixed

File: spec/rest/verix/verix_metadata_spec.rb [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+  describe '#list' do
+    subject(:list) { FinApps::REST::VerixMetadata.new(api_client).list }
+
+    it_behaves_like 'an API request'
+    it_behaves_like 'a successful request'
+    it('returns the verix environment') { expect(list[RESULTS]).to have_key(:environment) }
+    it('returns the verix client id') { expect(list[RESULTS]).to have_key(:client_id) }
+  end
+end
```
[the remark this answers is not in the document]
Comment (qbantek) [reply spec/rest/verix/verix_metadata_spec.rb]: Fixed

File: lib/finapps/rest/verix/verix_metadata.rb [hunk 9]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module FinApps
+  module REST
+    class VerixMetadata < FinAppsCore::REST::Resources # :nodoc:
+      def list
+        super 'v/metadata'
+      end
```
Comment (qbantek) [inline lib/finapps/rest/verix/verix_metadata.rb]: I would be more semantically correct to use the `show` method. Same pattern as in [Tenant Settings](https://github.com/finapps/ruby-client/blob/develop/lib/finapps/rest/tenant_settings.rb#L8).

Comment (sebeth-fa) [reply lib/finapps/rest/verix/verix_metadata.rb]: Fixed

File: lib/finapps/rest/verix/verix_metadata.rb [hunk 10]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module FinApps
+  module REST
+    class VerixMetadata < FinAppsCore::REST::Resources # :nodoc:
+      def show
+        super nil,'v/metadata'
```
[the remark this answers is not in the document]
Comment (sebeth-fa) [reply lib/finapps/rest/verix/verix_metadata.rb]: Fixed

[the remark this answers is not in the document]
Comment (qbantek) [reply lib/finapps/rest/verix/verix_metadata.rb]: Perfect

**Pull request merged**